views/js/pciCreator/ims/mathEntryInteraction/runtime/mathquill/mathquill.css

```
@@ -17,7 +35,7 @@
   display: inline-block;
 }
 .mq-editable-field .mq-cursor {
-  border-left: 1px solid black;
+  border-left: 1px solid currentColor;
```

Stylelint (17.9.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)